### PR TITLE
[MBL-16021][Teacher] - Implement Teacher FilesE2E test

### PR DESCRIPTION
**Pull request by kdeakinstructure**
Extending Page object with FileListPage.
Adding ID to fragment_file_list.xml
Extract pressBackButton() method to ViewUtils.

refs: MBL-16021
affects: Teacher
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/teacher/src/androidTest/java/com/instructure/teacher/ui/pages/AssignmentSubmissionListPage.kt [hunk 1]
```diff
@@ -163,4 +177,29 @@ class AssignmentSubmissionListPage : BasePage() {
...
         waitForViewWithText(android.R.string.ok).click()
     }
+
+    fun assertFileDisplayed(fileName: String) {
+        val matcher =
+            Matchers.allOf(ViewMatchers.withId(R.id.fileNameText), ViewMatchers.withText(fileName))
+        Espresso.onView(matcher).assertDisplayed()
+    }
+
+    fun assertCommentAttachmentDisplayedCommon(fileName: String, displayName: String, click:Boolean = false) {
```
Comment (tamaskozmer) [inline apps/teacher/src/androidTest/java/com/instructure/teacher/ui/pages/AssignmentSubmissionListPage.kt:187]: The `click` parameter is unused

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**